docs/vsphere-golden-image.md

```diff
@@ -16,6 +16,7 @@ the patch is not available.*
 
 To start with, create a new virtual machine in vSphere from the selected Windows Server distribution using the ISO image.
 See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html) for installation guidelines.
+The supported [virtual hardware version](https://kb.vmware.com/s/article/1003746) is `15`.
```

Comment:
My info might be out of date, but I thought we were going to link to the [OpenShift docs](https://docs.openshift.com/container-platform/4.9/updating/updating-hardware-on-nodes-running-on-vsphere.html) for this information?

This should be OpenShift release specific and 4.9 supports 13+?

Reply:
The 4.9 docs have a note:
```
virtual hardware version 13 is now deprecated. These versions are still fully supported, but support will be removed in a future version of OpenShift Container Platform. Hardware version 15 is now the default
```
But yes, I agree, I'll change this point to OpenShift docs and not mention a any specific version here.

Reply:
Wow thats confusing as it contradicts the first paragraph of that page :laughing: